doc/apiv3.apib

```diff
@@ -157,12 +154,330 @@ he is allowed to see his own time entries in projects where the costs module is
+| costType    | The type of this entry                      | CostType      | not null              | READ                 |
+| user        | The user logging this entry                 | User          | not null              | READ                 |
+| workPackage | The work package that got the entry logged  | WorkPackage   | not null              | READ                 |
+
+## Properties
+| Property    | Description                                 | Type        | Constraints | Supported operations | Condition                   |
+| :---------: | ------------------------------------------- | ----------- | ----------- | -------------------- | --------------------------- |
+| id          | cost entry id                               | Integer     | x > 0       | READ                 |                             |
+| spentUnits  | The amount of units logged in this entry    | Float       |             | READ                 |                             |
+| spentOn     | The date when the units were spent          | Date        |             | READ                 |                             |
+| createdAt   | Time of creation                            | DateTime    |             | READ                 |                             |
+| updatedAt   | Time of the most recent change to the entry | DateTime    |             | READ                 |                             |
+
+## Cost Entry [/api/v3/cost_entries/{id}]
+
++ Model
+    + Body
+
+            {
+                "_type": "CostEntry",
+                "_links": {
+                    "self": {
+                        "href": "/api/v3/cost_entries/1"
+                    },
+                    "project": {
+                        "href": "/api/v3/projects/1"
+                    },
+                    "costType": {
+                        "href": "/api/v3/cost_types/1"
+                    },
+                    "user": {
+                        "href": "/api/v3/users/1"
+                    },
+                    "workPackage": {
+                        "href": "/api/v3/work_packages/1"
+                    }
+                },
+                "id": 1,
+                "spentUnits": 3.14,
+                "spentOn": "2015-03-31",
+                "createdAt": "2015-03-31T08:51:20Z",
+                "updatedAt": "2015-03-31T08:51:20Z"
+            }
+
+
+## view Cost Entry [GET]
+
++ Parameters
+    + id (required, integer, `1`) ... Cost Entry id
+
++ Response 200 (application/hal+json)
+
+    [Cost Entry][]
+
++ Response 403 (application/hal+json)
+
+    Returned if the client does not have sufficient permissions.
+
+    **Required permission:** view cost entries **or** view own cost entries (on cost entry's project)
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:MissingPermission",
+                "message": "You are not allowed to see this cost entry."
```

Comment:
The same answer like all the times before: Messages here are only examples of an ideal state.

All messages are good as long as they fullfill the criteria defined [here](https://www.openproject.org/api):

>* The `message` contains a human readable concise message describing the error
>    * It *optionally* includes specific information, for example which permission would have been needed to perform an action
>    * It is localized depending on the users preferences
>    * It *must not* include HTML or other kind of markup
>    * Error messages form complete sentences including punctuation